### ISSUE TITLE
Quantity issue with 4x6 prints

### ISSUE THREAD

**Issue by kristinahavlovic** [printdotio/printio-android-sdk#160]
Hi guys,

A customer placed two orders for 210 different prints, each in qty of 1, but received 8 different photos in large quantities. The issue is that when you look in the Images tab in Admin, you can see all of the 210 different photos, but in Summary tab only 8 of them are shown, each in random quantity (25, 42, 53 etc)

Here are the orders:
https://admin.print.io/#/orders/398880/summary
https://admin.print.io/#/orders/398879/summary

Please advise. 
Thanks!

@bomberaya 
@micahasmith

**Comment by Ivan1985** [comment]
@kristinahavlovic I'll try to recreate this issue and after that I can give you more info

**Comment by bomberaya** [comment]
@kristinahavlovic This seems like a vendor/integration issue. If you look at the orders, the customer's shopping cart had 8 items with different number of prints, and the vendor only printed the first print of each item (as many times as there are prints in the item).
@stevanjoncic Can you take a look at this?

**Comment by Ivan1985** [comment]
@kristinahavlovic @bomberaya @micahasmith @tobiasfrier  I couldn't recreate this issue, I tried to recreate it on few versions of SDK (3.1.7, 3.1.22, 3.1.28 and 3.1.39). I tried with lot of use cases with different photo sources and nothing happened, everything works normal/good.

**Comment by mkontic** [comment]
Hi guys, I think that we already had the same issue a few months ago. It was the old version of the app and somehow different images were "grouped" under one item. Not sure how, but I think this was our bug.

**Comment by bomberaya** [comment]
@mkontic Yes, but we are unable to reproduce this on any builds that Ivan mentioned (and these include the exact version that this order was made with). We are not sure how the customer managed to produce this order...

**Comment by mkontic** [comment]
So, if I understood correctly, we don't know what the bug is or how to fix it, right? :)

**Comment by bomberaya** [comment]
You understood correctly :) We can investigate some more, but being unable to actually reproduce the issue using the exact same version, we can't really be sure that we fixed it...

As far as the customer is concerned, I'm sure you can modify the order manually to produce the wanted order and ship it to the customer, correct?

**Comment by mkontic** [comment]
Unfortunately, we can't. Submitting new, corrected order would mean that we need to download 210 images that customer submitted and place the new order. However, we can easily create a coupon for customer to place new order and advise him to use website or update the app. From what I can remember, this issue was connected only to android 118 version (not sure how you call it). I think it was fixed in newer versions of our app. Which one is Hellppics latest version?

**Comment by bomberaya** [comment]
I understand :/

You can see the exact version of the PIO SDK in the order summary, under Meta Info (in this case SDKVersion: 3.1.22).
It is the latest HelloPics version available.

But take a look at the order that I just made using the exact same version of HelloPics:
https://admin.print.io/#/orders/405064/summary

In it, each photo is its own item (and I intentionally changed the quantity of third image to 2) - everything is OK - same HelloPics version...

**Comment by mkontic** [comment]
Yeah, I have no idea why is it happening sometimes. Have you tried adding images in "batches". For example, add 20 images, choose the size then go back and add 15 more? Because I can't find any logic explanation why and how images get grouped together..
